kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-weblogic-sample-domain-job-template.yaml

```diff
@@ -1,264 +1,5 @@
 # Copyright 2017, 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
Don't include 2017 in any of your copyrights?

Reply:
Sure